File: backend/ruby_foundations/exercises/small_problems_101/easy_7/capitalize_words.rb
def word_cap(string)
  capitalize_words = string.split(' ').map { |word| word.capitalize }
  capitalize_words.join(' ')
end

puts word_cap('four score and seven') == 'Four Score And Seven'
puts word_cap('the javaScript language') == 'The Javascript Language'
puts word_cap('this is a "quoted" word') == 'This Is A "quoted" Word'

File: frontend/javascript/exercises/string_and_text_processing/capitalize_words.js
function wordCap(sentence) {
  return sentence.split(' ').map(function (word) {
    return word[0].toUpperCase() + word.slice(1).toLowerCase();
  }).join(' ');
}

console.log(wordCap('four score and seven') === 'Four Score And Seven');    // 'Four Score And Seven'
console.log(wordCap('the javaScript language') === 'The Javascript Language'); // 'The Javascript Language'
console.log(wordCap('this is a "quoted" word') === 'This Is A "quoted" Word'); // 'This Is A "quoted" Word'